src/core_plugins/tag_cloud/public/tag_cloud.js

```diff
@@ -0,0 +1,250 @@
+import d3 from 'd3';
+import d3TagCloud from 'd3-cloud';
+import vislibComponentsSeedColorsProvider from 'ui/vislib/components/color/seed_colors';
+
+
+const ORIENTATIONS = {
+  'single': (tag) => 0,
+  'rightAngled': (tag) =>~~(Math.random() * 2) * 90,
+  'multi': (tag) => (~~(Math.random() * 12) * 15) - 90
+};
```

Comment:
Minor formatting improvements:

```javascript
const ORIENTATIONS = {
   single: () => 0,
   rightAngled: () => ~~(Math.random() * 2) * 90,
   multi: () => (~~(Math.random() * 12) * 15) - 90,
 };
```

It would also be nice to extract these into their own module and add unit tests documenting their behavior.